odl_video/settings.py

```diff
@@ -73,11 +75,31 @@ def get_var(name, default):
     )
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = False
 
 ALLOWED_HOSTS = ['*']
 
 
+WEBPACK_LOADER = {
+    'DEFAULT': {
+        'CACHE': not DEBUG,
+        'BUNDLE_DIR_NAME': 'bundles/',
+        'STATS_FILE': os.path.join(BASE_DIR, 'webpack-stats.json'),
+        'POLL_INTERVAL': 0.1,
+        'TIMEOUT': None,
+        'IGNORE': [
+            r'.+\.hot-update\.+',
+            r'.+\.js\.map'
+        ]
+    }
+}
+
+# Request files from the webpack dev server
+USE_WEBPACK_DEV_SERVER = os.environ.get('ODL_VIDEO_SERVICE_USE_WEBPACK_DEV_SERVER', False)
```

Comment:
We should use `get_var` here, it will parse the environment variable string into a boolean